packages/devtools-reps/src/reps/rep-utils.js

```diff
@@ -387,13 +387,30 @@ function isURL(token) {
   }
 }
 
+/**
+ * Returns a new array in which `char` are interleaved between the original items.
+ *
+ * @param {Array} items
+ * @param {String} char
+ * @returns Array
+ */
+function interleave(items, char) {
+  return items.reduce((res, item, index) => {
+    if (index !== items.length - 1) {
+      return res.concat(item, char);
+    }
+    return res.concat(item);
+  }, []);
+}
```

Comment:
This way works, I've seen `zip` used here `zip(items, delimiters)` where del is one less than items.